py-utils/src/utils/msg_bus/message_bus/analytical_consumer.py

```diff
@@ -0,0 +1,21 @@
+from message_bus import Consumer
+from message_bus.utils import add, check_topic, last_one_hour_data
+
+class AnalyticalConsumer(Consumer):
```

Comment:
Please move this into separate patch. Not a phase 1 deliverable.

Reply:
Moved